api/src/main/java/edu/cornell/mannlib/vitro/webapp/web/templatemodels/individual/GroupedPropertyList.java

```diff
@@ -67,7 +71,7 @@ is to make the get methods (getProperty and getPropertyAndRemoveFromList) method
     private List<PropertyGroupTemplateModel> groups;
 
     GroupedPropertyList(Individual subject, VitroRequest vreq,
-            boolean editing) {
+            boolean editing) throws Exception {
```

Comment:
Not a good idea to add a generic exception here. Should at least be limited to throwing the exceptions that occur in the method, but also consider whether they can be handled within the method and not changing the signature.